dvc/command/gc.py

```diff
@@ -79,12 +79,13 @@ def add_parser(subparsers, parent_parser):
         default=False,
         help="Keep data files for all git tags.",
     )
-    gc_parser.add_argument(
-        "--all-commits",
-        action="store_true",
-        default=False,
-        help="Keep data files for all commits.",
-    )
+    # Issue number 2681
+    # gc_parser.add_argument(
+    #     "--all-commits",
+    #     action="store_true",
+    #     default=False,
+    #     help="Keep data files for all commits.",
+    # )
```

Comment:
Hi @sagunji !

Thanks for the PR!

The proper solution for this https://github.com/iterative/dvc.org/issues/730#issuecomment-545739625 is to hide this option like so:

```
gc_parser.add_argument(
    "--all-commits",
    action="store_true",
    default=False,
    help=argparse.SUPPRESS,
)
```

Reply:
Hi @efiop 
I have pushed changes according to your request.

Thank you!